lib/Badge.js

```diff
@@ -42,7 +44,8 @@ export default class Badge extends Component {
         justifyContent: "center",
         backgroundColor: "#F50057",
         zIndex: 9999,
-        bottom: 10,
+        top: 3,
+        right: 27,
```

Comment:
In an active Tab, this position causes the Badge to be far away from the Icon. This happens because it's a fixed value, but the position depends on the screen dimension, shifting, and Tab amount.

![image](https://user-images.githubusercontent.com/4227520/36077552-9af4ff1a-0f6c-11e8-96f1-d01774299110.png)


I would suggest something like `left: '50%', marginLeft: 15`. This way, it's positioned in the center of the Tab and then moved to the left. Looks like this:

![image](https://user-images.githubusercontent.com/4227520/36077565-cbff6244-0f6c-11e8-9bff-a286d82ab5d8.png)

Feel free to use a bigger/smaller value for `marginLeft` – depending on how you think it should be positioned.

Reply:
done